docs/LanguageCode.md

```diff
@@ -0,0 +1,20 @@
+# LanguageCode
+
+- `v::languageCode()`
+
+Validates a language code based on ISO 639:
+
+```php
+v::languageCode()->validate('pt'); // true
+v::languageCode()->validate('en'); // true
+v::languageCode()->validate('it'); // true
+v::languageCode(LanguageCode::ALPHA3)->validate('ita'); // true
```

Comment:
I think that would be better just use the strings on the rule's constructor, like `"alpha-3"` instead. That's make a lot easy to use the rule and it's the best example we can give.